anta/inventory/__init__.py

```diff
@@ -246,29 +251,31 @@ def parse(
     # GET methods
     ###########################################################################
 
-    def get_inventory(self, *, established_only: bool = False, tags: list[str] | None = None) -> AntaInventory:
+    def get_inventory(self, *, established_only: bool = False, tags: list[str] | None = None, devices: list[str] | None = None) -> AntaInventory:
         """Return a filtered inventory.
 
         Args:
         ----
-            established_only: Whether or not to include only established devices. Default False.
+            established_only: Whether or not to include only established devices.
             tags: List of tags to filter devices.
+            devices: List of names to filer devices.
 
         Returns
         -------
-            AntaInventory: An inventory with filtered AntaDevice objects.
-
+            An inventory with filtered AntaDevice objects.
         """
 
         def _filter_devices(device: AntaDevice) -> bool:
-            """Select the devices based on the input tags and the requirement for an established connection."""
+            """Select the devices based on the inputs `tags`, `devices` and `established_only`."""
             if tags is not None and all(tag not in tags for tag in device.tags):
                 return False
-            return bool(not established_only or device.established)
+            if devices is None or len(devices) == 0 or device.name in devices:
```

Comment:
if we have an empty list would we not consider it rather as no devices? How could we get an empty list otherwise?